tests/contrib/django/test_django.py

```diff
@@ -1702,7 +1702,7 @@ def eval_lazy(lo):
 
     request = request_cls()
     request.path = request_path
-    request.META = {"HTTP_HOST": http_host}
+    request.set_tags({"HTTP_HOST": http_host})
```

Comment:
This looks like a find/replace mistake?

Reply:
It is. I'll make another pass to make sure I didn't miss anything else